Tasks/GitHubReleaseV0/main.ts

```diff
@@ -102,9 +102,12 @@ class Main {
         const releaseNoteInput = tl.getInput(Inputs.releaseNotes);
         const showChangeLog: boolean = tl.getBoolInput(Inputs.addChangeLog);
 
+        const compareWithRelease = tl.getInput(Inputs.compareWith);
+        showChangeLog && Utility.validateStartCommitSpecification(compareWithRelease);
+        const releaseTag = tl.getInput(Inputs.releaseTag);
```

Comment:
this input may not be set in all conditions. what does this return in that case?

Reply:
you should explicitly read for releaseTag only when comparewithRelease is the tag option else set it to empty